OpenFeature/build.gradle.kts

```diff
@@ -48,16 +45,53 @@ dependencies {
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.7.1")
 }
 
-publishing {
-    publications {
-        register<MavenPublication>("release") {
-            groupId = "dev.openfeature"
-            artifactId = "kotlin-sdk"
-            version = "0.0.1-SNAPSHOT"
+afterEvaluate {
+    publishing {
+        publications {
+            register<MavenPublication>("release") {
+                groupId = "dev.openfeature"
+                artifactId = "kotlin-sdk"
+                version = "0.0.1-SNAPSHOT"
```

Comment:
Is this correct? shouldn't we pull this from somewhere? The tag info for example?

Reply:
Yeah, sounds good. Just a note that we have the version duplicated both here and in `android { defaultConfig {} }`

Reply:
+1 on syncing Github release versions with Maven versions, once we get closer to an official Maven distribution